sdcm/logcollector.py

```diff
@@ -220,7 +220,8 @@ class MonitoringStack(BaseLogEntity):
 
     @staticmethod
     def get_monitoring_base_dir(node):
-        return node.parent_cluster.monitor_install_path_base
+        from sdcm.cluster import BaseMonitorSet
```

Comment:
please make it a fallback, if parent_cluster is None or doesn't exist.

Reply:
done